src/components/reports/ReportForm.test.tsx

```diff
@@ -32,9 +37,7 @@ describe("Report form", () => {
 
 
     // Assert
-    const expectedValue = `"mrn": "${newMRNValue}"`
     const result = await screen.findByRole('alert')
     expect(result).toBeInTheDocument();
```

Comment:
Is it sufficient to only detect that the text box is rendered? Are any other tests/content validations needed here do you think?

Reply:
Yeah I was wondering about this. Ideally we'd clear the FHIR API of all entities and make sure that they exist for a full end-to-end test - though this felt like a decent amount of work so wanted to at least have defined all of the FHIR resources first so that we didn't have to change it down the line. 

Do you think that method of testing would be better - in the long term we wouldn't show the bundle to the user, but this seemed like a lightweight way to ensure that a bundle has been submitted

Reply:
I think this should suffice for now as it will be removed at a later date, so no need to spend too much time on it.